src/uniparc/components/landing-page/LandingPage.tsx

```diff
@@ -0,0 +1,292 @@
+            unique sequence only once and giving it a stable and unique
+            identifier (UPI) making it possible to identify the same protein
+            from different source databases. A UPI is never removed, changed or
+            reassigned. UniParc contains only protein sequences and
+            cross-references. All other information about the protein must be
+            retrieved from the source databases using the database
+            cross-references. UniParc tracks sequence changes in the source
+            databases and archives the history of all changes. UniParc has
+            combined many databases into one at the sequence level and searching
+            UniParc is equivalent to searching many databases simultaneously.
+          </p>
+          <p>
+            <Link
+              to={{
+                pathname: LocationToPath[Location.UniParcResults],
+                search: stringifyQuery({ query: '*' }),
+              }}
+            >
+              Start searching in UniParc »
+            </Link>
+          </p>
+        </div>
+
+        {/* Table */}
+        <section
+          className={cn(
+            'uniprot-grid-cell--small-span-12',
+            `uniprot-grid-cell--medium-span-${expanded ? '12' : '4'}`
+          )}
+        >
+          <h2>Databases</h2>
+          <div className={styles['table-container']}>
+            {groupedDatabases.map((databases, index) => (
+              // eslint-disable-next-line react/no-array-index-key
+              <table key={index}>
+                <thead>
+                  <tr>
+                    <th>Cross-reference</th>
+                    <th>
+                      Number of
+                      <br />
+                      UniParc entries
+                    </th>
+                  </tr>
+                </thead>
+                <tbody>
+                  {databases.map(({ count, value, label }) => (
+                    <tr key={value}>
+                      <td>
+                        <Link
+                          to={{
+                            pathname: LocationToPath[Location.UniParcResults],
+                            search: stringifyQuery({
+                              query: '*',
+                              facets: `${FacetsEnum.Database}:${value}`,
+                            }),
+                          }}
+                        >
+                          {label || value}
+                        </Link>
+                      </td>
+                      <td className={styles.end}>
+                        <LongNumber>{count}</LongNumber>
+                      </td>
+                    </tr>
+                  ))}
+                </tbody>
+              </table>
+            ))}
+          </div>
+
+          <Button
+            variant="tertiary"
+            onClick={() => setExpanded((expanded) => !expanded)}
+          >
+            {expanded ? 'View less' : 'View more'} databases
```

Comment:
```suggestion
            {expanded ? 'View fewer' : 'View more'} databases
```
I only learnt recently about less vs fewer the other day. Less is for uncountable things while fewer is for countable things. "Less milk, fewer cookies."

Reply:
Ah, right, so we should also probably do that change to the other places where we have that more/less pattern (I think in the ExpandableMessage in Franklin)

Reply:
Good point. Basically:
```
diff --git a/src/components/expandable-list.tsx b/src/components/expandable-list.tsx
index ab60aa40..9fb07d53 100644
--- a/src/components/expandable-list.tsx
+++ b/src/components/expandable-list.tsx
@@ -20,7 +20,7 @@ export const ExpandableMessage: FC<ExpandableMessageProps> = ({
   showHideWording,
   nHiddenItems,
 }) => {
-  let message = `${showHideWording ? 'Hide' : 'Less'} ${descriptionString}`;
+  let message = `${showHideWording ? 'Hide' : 'Fewer'} ${descriptionString}`;
   if (!expanded) {
     if (nHiddenItems === undefined) {
       message = showHideWording ? 'Show' : 'More';
```
I'm happy to push and release if this works for you?

Reply:
Yep, thanks!

Reply:
Done!